Automattic/wp-calypso
# Jetpack Cloud: Add golden ticket dialog

## Proposed Changes

This PR adds a dialog to let a user accept a golden ticket.

## Testing Instructions



In Jetpack cloud, visit `/golden-token`. Check the styling, copy, animations, etc. The business logic will be added in a later PR.

## Pre-merge Checklist
